contracts/src/ethereum/TokenBridge.sol

```diff
@@ -71,6 +71,9 @@ contract TokenBridge is
     // The selector of the "handle_deposit" l1_handler on L2.
     uint256 constant DEPOSIT_HANDLER =
         1285101517810983806491589552491143496277809242732141897358598292095611420389;
+    // The selector of the "handle_rewards_update" l1_handler on L2.
```

Comment:
can we please move the declaration of our constants to the top?

Reply:
I agree, we followed [this implementation](https://github.com/starkware-libs/starkgate-contracts/blob/main/src/starkware/starknet/apps/starkgate/solidity/StarknetTokenBridge.sol) where constants are in the middle of the file. I also prefer putting them at the top.